catroid/src/main/java/org/catrobat/catroid/formulaeditor/FormulaElement.java

```diff
@@ -989,7 +989,7 @@ public boolean containsElement(ElementType elementType) {
 
 	public boolean isUserVariableWithTypeString(Sprite sprite) {
 		if (type == ElementType.USER_VARIABLE) {
-			DataContainer userVariableContainer = ProjectManager.getInstance().getCurrentlyPlayingScene()
+			DataContainer userVariableContainer = ProjectManager.getInstance().getCurrentlyEditedScene()
```

Comment:
why did you change this? are you sure we should use the `currentlyEditedScene` here? this is going to change how this behaves.

Reply:
since this method "isUserVariableWithTypeString(Sprite sprite)" is just only called from the method "getResultForComputeDialog(Context context)" , so nothing related to stage is changed. so I'm sure this is right.

Reply:
alright, thank you for clearing that up. :wink: